### DISCUSSION TITLE
Electric rate

### DISCUSSION THREAD

**Discussion by emelbardis** [OctoFarm/OctoFarm#1304] [Q&A]
Where do I input my electric rate for print cost calculations?
Also, what is 'maintenance' cost looking at and is this a user customizable number?

Thanks

![image](https://user-images.githubusercontent.com/20196908/194602759-31a9fc57-a445-418f-bac6-812fd25c70cc.png)

**Comment by NotExpectedYet** [comment]
> Where do I input my electric rate for print cost calculations? Also, what is 'maintenance' cost looking at and is this a user customizable number?
> 
> Thanks
> 
> ![image](https://user-images.githubusercontent.com/20196908/194602759-31a9fc57-a445-418f-bac6-812fd25c70cc.png)

Those are in printer settings 

If you go into "Printers" menu, then on the table with your printers theres a blue button on each one called "Manage" the dropdown will have an option called "Settings". Theres a tab in there called "Cost" and you input those into there. It replicates the Cost Plugins functionality.

**Reply by NotExpectedYet** [reply]
It will pick up the settings entered into this plugin but it's only on first scan. 

https://plugins.octoprint.org/plugins/costestimation/

**Reply by emelbardis** [reply]
Very nice. Thank you.
I love that most of your buttons are redundant across screens so you can navigate wherever you want to go from just about anywhere without backing out. I somehow missed that button as I've only been using OctoFarm for about a week. It's so great!!

**Reply by NotExpectedYet** [reply]
Thanks for the praise. 

Yeah i try to keep clicks to a minimum on the interface. It's showing its age now though aha. Would you mind marking this as answered please just to alert other users who might have a similar question.
